src/Core/SmsChannel.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace EscolaLms\TemplatesSms\Core;
+
+use Aloha\Twilio\Twilio;
+use EscolaLms\Templates\Contracts\TemplateChannelContract;
+use EscolaLms\Templates\Core\AbstractTemplateChannelClass;
+use EscolaLms\Templates\Core\TemplateSectionSchema;
+use EscolaLms\Templates\Enums\TemplateSectionTypeEnum;
+use EscolaLms\Templates\Events\EventWrapper;
+use EscolaLms\Templates\Models\Template;
+use EscolaLms\Templates\Models\TemplateSection;
+use Illuminate\Support\Collection;
+use EscolaLms\Core\Models\User;
+
+class SmsChannel extends AbstractTemplateChannelClass implements TemplateChannelContract
+{
+    public static function send(EventWrapper $event, array $sections): bool
+    {
+        $user = $event->user();
+
+        $channelEnabled =
+            isset($user->notification_channels)
+            && array_search(SmsChannel::class, json_decode($user->notification_channels));
+
+        if (!$channelEnabled || !$user->phone) {
+            return false;
+        }
+
+        $sid = config('escolalms_templates_sms.twilio.sid');
+        $token = config('escolalms_templates_sms.twilio.token');
+        $from = config('escolalms_templates_sms.twilio.from');
+
+        $twilio = new Twilio($sid, $token, $from);
+        $twilio->message($user->phone, $sections['content']);
```

Comment:
what if we want another provider here, like sms api
IMHO this should be agnostic to framework 

I see some scenariosc here 
* This should send Laravel notification that will trigger specific method (smsapi, twilio, netxo, whatever) 
* we can use dependecy injection and bind specific method 

But 4sure twilio cannot be called directly here

Reply:
The SMS provider should be set in settings, and the whole configuration can be done similar to laravel with disks